code/modules/projectiles/ammo_datums.dm

```diff
@@ -961,6 +961,28 @@
 	stamina_damage = 10
 	shrapnel_chance = 0
 
+/datum/ammo/bullet/smg/mp27
+	name = "simple submachinegun bullet"
+	damage = 50
+	accurate_range = 5
+	effective_range_max = 6
+	penetration = 0
+	shell_speed = AMMO_SPEED_TIER_6
+	damage_falloff = DAMAGE_FALLOFF_TIER_6
+	scatter = SCATTER_AMOUNT_TIER_6
+	accuracy = HIT_ACCURACY_TIER_2
+
+// less damage than the m39, but better falloff, range, and AP
+
+/datum/ammo/bullet/smg/ppsh
+	name = "crude submachinegun bullet"
+	damage = 28
```

Comment:
```suggestion
	damage = 24
```

Reply:
this makes the gun unusably weak - see clips in linked PR's discussion

Reply:
compromised to 26